pluralsight-projects/React-QuizComponent
# #160870148 Project setup

### What does this PR do?
 Create the Quiz Component

### Description of Task to be completed?
Create the Quiz Component

### How should this be manually tested?
- fetch this branch with `git fetch origin ch-project-setup-160870148:ch-project-setup-160870148`
- Checkout to the branch
- Run `npm install`
- Run `npm start`
- Then a browser should open showing a blank page

### Any background context you want to provide?
Na


### What are the relevant pivotal tracker stories?
[#160870148](https://www.pivotaltracker.com/n/projects/2200608/stories/160870148)

### Screenshots (if appropriate)
NA
